src/services/table/FilterDropdown.tsx

```diff
@@ -0,0 +1,66 @@
+import { SearchOutlined } from '@ant-design/icons';
+import { Button, Input, Space } from 'antd';
+import { FilterConfirmProps } from 'antd/lib/table/interface';
+import React, { MutableRefObject } from 'react';
+
+import style from './FilterDropdown.module.scss';
+
+export default function FilterDropdown({
+  dataIndex,
+  setSelectedKeys,
+  selectedKeys,
+  confirm,
+  clearFilters,
+  searchInput,
+  handleSearchCallback,
+  handleResetCallback,
+}: {
+  dataIndex: string;
```

Comment:
I recommend move definition of props into separate file. It's hard to read code now.